pkg/apis/pipeline/v1alpha1/pull_request_resource.go

```diff
@@ -104,10 +109,13 @@ func (s *PullRequestResource) GetOutputTaskModifier(ts *TaskSpec, sourcePath str
 
 func (s *PullRequestResource) getSteps(mode string, sourcePath string) []Step {
 	args := []string{"-url", s.URL, "-path", sourcePath, "-mode", mode}
+	if s.Provider != "" {
+		args = append(args, []string{"-provider", s.Provider}...)
+	}
 
 	evs := []corev1.EnvVar{}
 	for _, sec := range s.Secrets {
-		if strings.EqualFold(sec.FieldName, githubTokenEnv) {
+		if strings.EqualFold(sec.FieldName, authTokenEnv) {
```

Comment:
There were recent changes around this. See https://github.com/tektoncd/pipeline/pull/1615

Reply:
Should be fixed.